.secrets.baseline

```diff
@@ -117,6 +117,15 @@
         "line_number": 20
       }
     ],
+    "infrastructure/lambda/public-api.yaml": [
```

Comment:
did we need to add any secrets for this?

Reply:
i have added the secret to reflect the new name in template file. Public api is using token from credential issuer.

Reply:
.secrets.baseline is getting generated by running `detect-secrets scan` all good